reconcile/utils/gql.py

```diff
@@ -229,29 +263,61 @@ def init(
     commit: Optional[str] = None,
     commit_timestamp: Optional[str] = None,
 ):
-    global _gqlapi
-    _gqlapi = GqlApi(
+    return GqlApiSingleton(
```

Comment:
This is not exactly match previous logic, previously it will always create a new `GqlApi`, while `__new__` is a conditional create, we need a different force reinit method.

Reply:
True, but why do we want to reinit? I removed cause I found it not required. The client will be initialized on every integration run (since GqlApi.close() is called)